vulcanai/models/basenetwork.py

```diff
@@ -51,7 +51,7 @@ class BaseNetwork(nn.Module):
         input_networks : list of BaseNetwork
             A network object provided as input.
         num_classes : int or None
-            The number of classes to predict.
+            The number of classes to predict. Must be at least 2.
```

Comment:
Would you be able to add more descriptions here on how a user would encode a binary outcome as 2 classes just so there's no confusion?

Reply:
done